### ISSUE TITLE
What is AudioSegment creating? 

### ISSUE THREAD

**Issue by TaoTeCha** [jiaaro/pydub#540]
I have a 500mb .mb4 file. When I try to convert to an Audiosegment it gives me an error that file can not be over 4GB. Is it decompressing the audio into something similar to .wav? Because I know a wav file of the same audio would be about 4.5GB. 

Also, is there any way to work with my 500mb mb4 file without manually splitting it into chunks before hand? 

Thank you.

**Comment by GreyAlien502** [comment]
Yes, the data is stored in pydub as uncompressed data, just like a .wav filekkk.
There is not currently any solution for processing large files with pydub.
